md/bonita-container.md

```diff
@@ -0,0 +1,111 @@
+# Deploy a Bonita platform using docker
+
+How to install and use the docker distribution of the Bonita platform.
+
+This guide assumes you use the subscription version of the docker image. For the community version, simply skip the _Retrieve a licence_ step.
+It also assumes you are on a unix-based operating system.
+
+## Quick start
```

Comment:
Does not display well....
![Capture d’écran de 2019-10-31 15-02-35](https://user-images.githubusercontent.com/4921958/67953509-8e05a180-fbef-11e9-98eb-bd76b98f946a.png)